.github/workflows/request-reviewer.yml

```
@@ -15,7 +15,7 @@ on:
 
 jobs:
   request-review:
-    if: (github.event.action == 'created' && github.event.issue.draft == false && github.event.comment.user.login == github.event.issue.user.login) || (github.event.action != 'created' && github.event.pull_request.draft == false)
+    if: (github.event.action == 'created' && github.event.issue.state == 'open' && github.event.issue.draft == false && github.event.comment.user.login == github.event.issue.user.login) || (github.event.action != 'created' && github.event.pull_request.state == 'open' && github.event.pull_request.draft == false )
```

I think this could maybe be simplified to:

```suggestion
    if: (github.event.issue.state == 'open' && github.event.issue.draft == false) && (github.event.action != 'created' || github.event.comment.user.login == github.event.issue.user.login)
```